test/unittest/test_utils.py

```diff
@@ -74,21 +74,13 @@ def test_open_for_write_file_exists(self):
         with mock.patch('io.open',
                         mock.mock_open(),
                         create=True) as mock_open, \
-             mock.patch('os.path.exists', return_value=True):
+             mock.patch.object(
+                utils.Path,
+                'mkdir',
+                return_value=True) as mock_create:
             utils._open_for_write(filename)
 
-            mock_open.assert_called_once_with(filename, 'w')
-
-    def test_open_for_write_file_does_not_exist(self):
```

Comment:
Can we keep this test.

Reply:
What does it accomplish? I removed it because when I modified it to use the new code it was literally the same code as the above test because there's no longer a branch.

Reply:
You are right, I was thinking that it would catch using the wrong parameters when making the directory, but we are mocking everything. 

In this case I think then we need to change the name of the test to just `test_open_for_write`

Reply:
Switched the name.